src/package_manager/apt.ts

```diff
@@ -62,6 +62,33 @@ const distributionSpecificAptDependencies = {
 		// libc++-dev and libc++abi-dev installs intentionally removed because:
 		// https://github.com/ros-tooling/setup-ros/issues/506
 	],
+	noble: [
+		// Basic development packages (from ROS 2 source/development setup instructions)
+		// ros-dev-tools includes many packages that we needed to include manually in Focal & older
+		"python3-flake8-docstrings",
+		"python3-pip",
+		"python3-pytest-cov",
+		"python3-flake8-blind-except",
+		"python3-flake8-builtins",
```

Comment:
If we are using `rosdep` to install other dependencies, this shouldn't be necessary anymore; they are already included in https://github.com/ament/ament_lint/blob/rolling/ament_flake8/package.xml now.

Reply:
I agree this should not be necessary. For now, it was just copied from jammy.